judgments/views/detail.py

```diff
@@ -78,10 +78,18 @@ def detail(request, judgment_uri):
 
     context = {}
 
-    if "press-summary" in judgment_uri:
```

Comment:
I know we previously said that we should allow for urls other than /1, but after discussing, we realise thats a proper edge case so will just always look for /1 until we are told otherwise.

Having it more general would have complicated things here unncessarily

Reply:
:+1: